src/README_OLD.md

```diff
@@ -0,0 +1,203 @@
+
+`npm install --save @chili-publish/editor-sdk@v0.0.1`
+
+To check our versions and tags, go to [our package on npmjs ](https://www.npmjs.com/package/@chili-publish/editor-sdk/v/0.47.1?activeTab=versions) or take [the title of our github release](https://github.com/chili-publish/editor-sdk/releases), this is always in sync with the tagname that is used.
+
+<br>
+
+#### Using the script tag
+
+Once you have received access to the SDK, you will receive a link the SDK javascript file that you need to include inside your html.
+
+We recommend including the script in a script tag, right before the closing tag of the body (`</body>`) but keep in mind that your integration script should come below the SDK inclusion.
+
+Example in `index.html`:
+
+```html
+<html>
+    <head>
+        {# all of your head metadata and stylesheets should come here #}
+    </head>
+    <body>
+        <h1>My first integration</h1>
+        {# all of your custom html #}
+        <script src="https://path/to/sdk/latest/main.js"></script>
+        <script src="integration.js"></script>
+    </body>
+</html>
+```
+
+In the integration script we then need to initialise the SDK with certain parameters and values.
+
+Example in `integration.js`:
+
+```javascript
+const sdk = new EditorSDK({
+    editorLink: 'https://link/to/the/editor', // Will be provided
+    editorId: 'chili-editor-example', // if not provided, default will be 'chili-editor'
+});
+```
+
+The SDK is now initialised and ready to be used.
+If you want to make the SDK available in the console you can bind the sdk property to the window, this can be handy, but this isn't necessary.
+
+Example in `integration.js`:
+
+```javascript
+window.SDK = sdk;
+
+// or it can be done in one go:
+window.SDK = new EditorSDK({
+    // ... see example above
+});
+```
+
+<br/>
+
+### Get an instance of the editor on your page
+
+As you will see, there is no editor yet in your application, therefore you'll need to give the SDK the command to load the editor dynamically in your html. The `editorId` you provided or the default one, will be used as a target of where the editor will be injected, so you need to create an (preferably) empty `<div>` with the same id as used for editorId or the default id.
+
+Example in `index.html`:
+
+```html
+{# all of the html above #}
+<h1>My first integration</h1>
+
+<div id="chili-editor-example"></div>
+
+{# all of your custom html #}
+
+<script src="https://path/to/sdk/latest/main.js"></script>
+{# all of the html below #}
+```
+
+The SDK should also be aware that there is now room in the html for injecting the editor, so you should call the `loadEditor` method in your integration script.
+
+Example in `integration.js`:
+
+```javascript
+// at the bottom
+window.SDK.loadEditor();
+```
+
+If you now refresh your page in the browser you will see And like magic an instance of the editor gets loaded on your very own page.
```

Comment:
something seems wrong in the wording

Reply:
old readme, not used anymore but is there still as a reference, should be deleted soon